core/vm/interpreter.go

```diff
@@ -133,6 +136,15 @@ func (in *Interpreter) Run(contract *Contract, input []byte) (ret []byte, err er
 	)
 	contract.Input = input
 
+	// Reclaim the stack as an int pool when the execution stops
+	defer func() {
+		in.intPool.put(stack.data...)
+
+		// Return the int pool to the pool
+		poolOfIntPools.put(in.intPool)
+		in.intPool = nil
+	}()
```

Comment:
There's no point in putting `in.intPool.put(stack.data...)` into the defer. There's no point to put a defer here.